src/input/Slider.module.css

```diff
@@ -0,0 +1,88 @@
+.slider[type="range"] {
+  -ms-appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+
+  background-color: transparent;
+
+  /* This (slider-color) is 50% transparent to allow the voice activity slider to
+     display volumebar behind the slider */
+  --slider-color: var(--quinary-content-50);
```

Comment:
How does this affect the other sliders?

Reply:
![image](https://user-images.githubusercontent.com/72016555/186922712-5f4b0057-94ca-4557-9880-4e0c03d1d7c5.png)

We can move this otherwise if it is a serious change.